### ISSUE TITLE
OSX permission denied

### ISSUE THREAD

**Issue by GeneBlue** [Cisco-Talos/GhIDA#8]
```
GhIDA:: [DEBUG] found 0 symbols
GhIDA:: [INFO] XML exporting completed
GhIDA:: [DEBUG] decompiled cache miss (20026418)
GhIDA:: [DEBUG] Decompiling 20026418
GhIDA:: [DEBUG] EXPORT_XML_FILE: False
GhIDA:: [!] [Errno 13] Permission denied
GhIDA:: [!] Ghidra headless analysis failed
GhIDA:: [!] Decompilation interrupted.
GhIDA:: [DEBUG] DisasmsHandler HELLO
GhIDA:: [DEBUG] EXPORT_XML_FILE: False
GhIDA:: [DEBUG] decompiled cache miss (20026418)
GhIDA:: [DEBUG] Decompiling 20026418
GhIDA:: [DEBUG] EXPORT_XML_FILE: False
GhIDA:: [!] [Errno 13] Permission denied
GhIDA:: [!] Ghidra headless analysis failed
GhIDA:: [!] Decompilation interrupted.
```

**Comment by wumb0** [comment]
/ghidra/ghidra_9.0.2/support/analyzeHeadless.bat . Temp -import /Users/bob/509383E505C973ED7534A06B3D19688D_qVouD.xml -scriptPath /Applications/IDA Pro 7.3/ida64.app/Contents/MacOS/plugins/ghida_plugin/ghidra_plugin -postScript FunctionDecompile.py 87420 /var/folders/y6/cmx92ln52sz11gt1d051qf1h0000gn/T/87420_TmUvoQ -noanalysis -deleteProject

ghida_plugin/config.py only checks for "linux" in sys.platform and not "darwin"
Line 59 should probably be:
```
if sys.platform in ['linux', 'darwin']:
```

**Comment by jimmy-sonny** [comment]
Please, if possible test https://github.com/Cisco-Talos/GhIDA/commit/ff020808ac728fa1826f26f8cca678bda66aa16a on OSX.

**Comment by jimmy-sonny** [comment]
Closing the issue. This has been further addressed in 66e03bc.

Thanks for the suggestion.
